bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -322,11 +322,6 @@ spec:
 
           exec /usr/bin/ovnkube \
             --init-master "${K8S_NODE}" \
-            --cluster-subnets "${OVN_NET_CIDR}" \
-            --k8s-service-cidr "${OVN_SVC_CIDR}" \
-            --k8s-apiserver "{{.K8S_APISERVER}}" \
-            --ovn-config-namespace openshift-ovn-kubernetes \
-            --nodeport \
```

Comment:
@pecameron don't we need a `--config-file=/run/ovnkube-config/ovnkube.conf` here too?

Reply:
You can also remove the `--logfile /dev/stdout` argument from this ovnkube command since it's now in the config file.

Reply:
@dcbw I added the --config. Looking at the code it appears to be the default, if it exists. I noticed and pushed similar changes before reading your comment above. --logfile will get removed in the next push.

Reply:
@pecameron there is a default config file, but that default path sure isn't /run/ovnkube-config/